alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/AbstractResourceActions.java

```diff
@@ -77,6 +84,7 @@ public AbstractResourceActions(DescriptorKey descriptorKey, ConfigContextEnum co
     @Override
     public final ActionResponse<M> getAll() {
         if (!authorizationManager.hasReadPermission(context.name(), descriptorKey.getUniversalKey())) {
+            logger.error(String.format(FORBIDDEN_ACTION_FORMAT, "Get all"));
```

Comment:
This should be logged at DEBUG. As should all of the other new logs in this class.